lib/beaker/dsl/helpers/puppet_helpers.rb

```diff
@@ -323,7 +323,7 @@ def bounce_service host, service, curl_retries = nil, port = nil
           curl_retries = 120 if curl_retries.nil?
           port = options[:puppetserver_port] if port.nil?
           if host.graceful_restarts?
-            apachectl_path = host.is_pe? ? "#{host['puppetsbindir']}/apache2ctl" : 'apache2ctl'
+            apachectl_path = host.is_pe? ? "#{host['puppetsbindir']}/apachectl" : 'apachectl'
```

Comment:
@ferglor I'm sorry for delaying this further, but this check does sound like a better idea. There's a [`host.check_for_command`](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host/unix/pkg.rb#L10-L22) method that would be useful for this.